### YamlMime:UniversalReference
api_name: []
items:
- fullName: msrest.exceptions.raise_with_traceback
  langs:
  - python
  module: msrest.exceptions
  name: raise_with_traceback(exception, message='', *args, **kwargs)
  source:
    id: raise_with_traceback
    path: msrest\exceptions.py
    remote:
      branch: master
      path: msrest\exceptions.py
      repo: https://github.com/Azure/msrest-for-python.git
    startLine: 36
  summary: 'Raise exception with a specified traceback.


    This MUST be called inside a "except" clause.'
  syntax:
    content: raise_with_traceback(exception, message='', *args, **kwargs)
    parameters:
    - description: Error type to be raised.
      id: exception
      type:
      - Exception
    - defaultValue: ''
      description: Message to include with error, empty by default.
      id: message
      type:
      - str
    - description: Any additional args to be included with exception.
      id: args
  type: function
  uid: msrest.exceptions.raise_with_traceback
references: []
